src/components/badge/badge.css

```diff
@@ -0,0 +1,57 @@
+@import url('/src/components/utils.css');
+@import url('/src/components/avatar/avatar.css');
+
+.badge-container{
+    position : relative ;
+}
+.badge-status {
+    position: absolute;
+    min-width: 1.8rem;
+    height: 1.8rem;
+    border-radius: 50%;
+    border: 2px solid var(--white-color);
+    right: 2.5rem;
+    bottom: 2.5rem;
+}
+
+.status-online {
+    background-color: var(--status-online);
+}
+.status-offline {
+    background-color: var(--status-offline);
+    right: 1em;
+}
+
+.status-away {
+    background-color: var(--status-away);
+    right: 1rem;
+    bottom : 1rem;
+}
+
+.badge-primary{
+    position : absolute ;
+    background-color : var(--ca-primary-color-lighter);
+    color : var(--white-color);
+    padding : 0.6rem ;
+    border-radius : 2rem ;
+    top : 10px ;
+    left : 610px ;
```

Comment:
You should work with these values as the badges flow toward right when the text content is lesser or overflows when text content is more. May be % values can help here.